inst/rmd/rnasum.Rmd

```diff
@@ -122,6 +123,11 @@ ref_dataset.list <- vector("list", length(dataset)) |> set_names(dataset)
 ref_genes.list <- RNAsum::get_refgenes(params)
 # sample WTS/WGS data
 sample_data.list <- RNAsum::read_sample_data(params, results_dir, tx2gene = tx2ensembl)
+# check counts input was correctly read and not null
```

Comment:
So the R session will first call `RNAsum::read_sample_data` which is an intensive and time-consuming process. If no salmon or kallisto inputs have been provided, it goes to waste. What if you catch that error at the beginning of `RNAsum::read_sample_data`?